src/app/content/highlights/components/Card.tsx

```diff
@@ -56,6 +59,22 @@ const Card = (props: CardProps) => {
   const locationFilters = useSelector(selectHighlights.highlightLocationFilters);
   const hasUnsavedHighlight = useSelector(selectHighlights.hasUnsavedHighlight);
 
+  const { isFocused, highlight: { id }, focus, shouldFocusCard } = props;
```

Comment:
so i think `isFocused` is really like.... "highlight or card are focused so the card should be displayed in its 'active' state where it is indented a little or is shown at all on smaller screens"

i think this is rightfully a different thing than your `shouldFocusCard` which indicates moving the actual browser focus into the card, but the nomenclature is confusing to me. maybe we should rename `isFocused` to `isActive` ? or `isSelected` ?